java/RSPEC-2157.java

```diff
@@ -0,0 +1,4 @@
+class Team implements Cloneable {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout comment separator -->
---
RSPEC-2176: Class has same name as interface
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._